Buttplug.Core/Messages/Messages.cs

```diff
@@ -177,7 +177,7 @@ public class DeviceMessageInfo
         /// </summary>
         [SuppressMessage("ReSharper", "MemberInitializerValueIgnored", Justification = "JSON.net doesn't use the constructor")]
         [JsonProperty(Required = Required.Always, NullValueHandling = NullValueHandling.Ignore)]
-        public Dictionary<string, MessageAttributes> DeviceMessages = new Dictionary<string, MessageAttributes>();

```

Comment:
This is needed to stop JSON.net from setting this to null if empty (something to do with the reflection based construction). There's a suppression annotation above for it